kubeadm: add instructions about rebalancing CoreDNS Pods after joining more nodes (#30657)
kubernetes/website: content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

A few seconds later, you should notice this node in the output from `kubectl get
nodes` when run on the control-plane node.

{{< note >}}
As the cluster nodes are usually initialized sequentially, the CoreDNS Pods are likely to 

> run on the first control-plane node entirely. If you install a cluster with more than one node, 

Reviewer: “entirely” in this context reads strange.

Author: Thanks @tengqm 
Updated!

> all run on the first control-plane node. If you install a cluster with more than one node, 
> we recommend that you rebalance the CoreDNS Pods with `kubectl -n kube-system rollout restart deployment coredns`.

Reviewer 2: nit ([avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we), and timing for when to do this):
```suggestion
rebalance the CoreDNS Pods with `kubectl -n kube-system rollout restart deployment coredns`
after you first add nodes that are not part of the control plane.
```

How about that?

Author: Thanks @sftim 
Updated.
